### DISCUSSION TITLE
Sudden account suspension for no apparent reason

### DISCUSSION THREAD

**Discussion by Kareem-dev** [community/community#128195] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey,

7 days ago (5th of June), my account got suspended for no apparent reason. I was just working normally but then I was randomly logged out, and when I tried to log back in, I found my account suspended. I didn't receive an email stating what I've done wrong, so this seems like a random suspension. This was just completely unexpected and it really messed up a lot of my work and my mental health. My username was @Kareem01001011. Please help me fix this issue as soon as possible, as this has taken too long and I really need my account before the end of this month.
Ticket ID: 2827874

Thank you for your time

**Comment by PerryCodes** [comment]
~~Dude.  What's confusing about your account getting suspended?  It certainly wasn't "for no apparent reason" when you've got a Spam Bot project..~~

**Reply by Kareem-dev** [reply]
I made it to troll my friends! I didn't have a sliver of ill intent when making/using it. But if this is the problem, I'll happily delete it out of existence. I just want my account back 😢.

**Reply by PerryCodes** [reply]
Yeah - I'm not sure. That screenshot getting posted to Reddit made me curious and I should have looked first before commenting above... there are TONS of other spam projects all over GitHub, so I doubt that was it.  I hope you at least get a reason for the account getting pulled.  That type of action without any warning or notification really sucks.

**Comment by ebndev** [comment, marked as answer]
Hi @Kareem-dev,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
